fdbserver/masterserver.actor.cpp

```diff
@@ -294,7 +294,9 @@ ACTOR Future<Void> newCommitProxies(Reference<MasterData> self, RecruitFromConfi
 		req.recoveryCount = self->cstate.myDBState.recoveryCount + 1;
 		req.recoveryTransactionVersion = self->recoveryTransactionVersion;
 		req.firstProxy = i == 0;
-		TraceEvent("CommitProxyReplies", self->dbgid).detail("WorkerID", recr.commitProxies[i].id());
+		TraceEvent("CommitProxyReplies", self->dbgid)
```

Comment:
I think this is better named as "InitializeCommitProxy", since there is no reply at this point.

Reply:
But if I rename this existing event name, I am afraid this may break our existing queries.

Reply:
SG. Didn't realize that it already exists.